src/Suggestions/Suggestions.module.css

```diff
@@ -0,0 +1,115 @@
+.suggestionSection {
+  padding: 112px 0;
+  margin: 0 auto;
+  max-width: 600px;
+  width: 100%;
+  box-sizing: border-box;
+}
+.suggestionSectionOnHomePage {
+  position: fixed;
+  top: 25%;
+  left: 65.45%;
+  height: 100vh;
+  width: 293px;
+}
+.text {
+  font-size: 16px;
+  font-weight: 600;
+  margin: 0;
+  margin-bottom: 6px;
+  margin-left: 13px;
+}
+.textOnHomePage {
+  color: #8e8e8e;
+  font-weight: 600;
+  margin-left: 14px;
+}
+.textViewButton {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.seeAllButton {
+  border: 0;
+  background: none;
+  font-weight: 600;
+  color: #262626;
+  padding: 0;
+  margin-right: 14px;
+  cursor: pointer;
+  text-decoration: none;
+  font-size: 12px;
+}
+.contentSection {
+  background-color: #fff;
+}
+
+.content {
+  display: flex;
+  padding: 8px 16px;
+  align-items: center;
+}
+.contentSectionOnHomePage {
+  padding: 0;
+  box-sizing: border-box;
+  display: flex;
+  padding: 8px 16px;
+  align-items: center;
+}
+.userNameAndReason {
+  flex: 1;
+}
+
+.userName {
+  margin: 0;
+  margin-left: 12px;
+  font-weight: 600;
+  text-decoration: none;
+  color: #262626;
+}
+.userFullName {
+  margin: 0;
+  margin-left: 12px;
+  margin-top: 2px;
+  color: #8e8e8e;
+  font-weight: 400;
+  font-size: 12px;
+  max-width: 195px;
+}
+.userReason {
+  margin: 0;
+  margin-left: 12px;
+  padding-top: 2px;
+  color: #8e8e8e;
+  font-weight: 400;
+  font-size: 12px;
+}
+.userReasonOnHomePage {
+  margin: 0;
+  margin-left: 12px;
+  margin-top: 0px;
+  color: #8e8e8e;
+  font-weight: 400;
+  font-size: 12px;
+  max-width: 165px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
```

Comment:
Same as above (`.userReason.onHomePage`). Even if the CSS is different, it's always better to use subclasses. If you decide to change all the buttons one day, you'll only change the main class.